### ISSUE TITLE
Need 1.4 Update

### ISSUE THREAD

**Issue by SunnySissi** [rheirman/GiddyUpCore#46]
Need 1.4 Update,thx

**Comment by jjerhub** [comment]
Hey @SunnySissi, just relaying here, on the Steam workshop page about an hour ago the author stated that they likely won't be updating this mod for 1.4 any time soon due to other personal priorities.

**Comment by jjerhub** [comment]
@rheirman I know on your steam workshop page for Core you said you are looking for a skilled modder to help bring this up-to-date now that Biotech has released.  While I by no means claim to be a skilled modder, I have been a software developer for over 10 years focusing in C# and may be interested in picking up the "GiddyUp" project.  You can email me at jjerhub@gmail.com.

**Comment by SunnySissi** [comment]
@jjerhub Actually I think you can go straight to version 1.4 of this mod, then mention the 1.4mod in the intro when the Steam workshop is released, and leave a comment under this mod's Steam workshop page and point to your new mod link.

**Comment by jjerhub** [comment]
> @jjerhub Actually I think you can go straight to develop 1.4 of this mod, then mention the 1.4mod in the intro when the Steam workshop is released, and leave a comment under this mod's Steam workshop page and point to your new mod link.

@SunnySissi Yes, that is possible, but per the author's description on the workshop page they have requested to not publish altered versions of the mod beyond things like translations.  Therefore my message here was an attempt to placate that request, but still perhaps find a way forward.

**Comment by rheirman** [comment]
@jjerhub 
Hi there. Thanks for offering help! I'll send you an email!

**Comment by rheirman** [comment]
> @jjerhub Actually I think you can go straight to develop 1.4 of this mod, then mention the 1.4mod in the intro when the Steam workshop is released, and leave a comment under this mod's Steam workshop page and point to your new mod link.

Updating the main mod through is preferred, because otherwise the Steam workshop gets flooded with unofficial versions that can potentially get out of date, or not be updated as well as I wish. I do understand people do this though if the mod author is not reachable.
